tests/phpunit/tests/l10n/getLocale.php

```diff
@@ -60,10 +56,6 @@ public function test_network_option_should_be_fallback_on_multisite() {
 	 * @group ms-excluded
 	 */
 	public function test_option_should_be_respected_on_nonmultisite() {
-		if ( is_multisite() ) {
-			$this->markTestSkipped( 'This test does not apply to Multisite.' );
-		}
```

Comment:
`@group` tags are all nice and dandy, but don't prevent a test from _not_ being run without complying with certain conditions. Only for running them selectively.

This condition should not be removed.